dll/win32/shell32/CQueryAssociations.cpp

```diff
@@ -597,13 +600,16 @@ HRESULT CQueryAssociations::GetCommand(const WCHAR *extra, WCHAR **command)
 
     if (ret)
     {
+        ERR("0x%lX\n", ret);
         return HRESULT_FROM_WIN32(ret);
     }
 
     if (!extra)
     {
         /* check for default verb */
         hr = this->GetValue(hkeyShell, NULL, (void**)&extra_from_reg, NULL);
+        if (FAILED(hr))
+            hr = this->GetValue(hkeyShell, L"open", (void**)&extra_from_reg, NULL);
```

Comment:
Why is this needed? The "open" verb is not part of the "default" one(s) that can be handled with the code below?

Reply:
`batfile` has `edit` command and `open` command. We had to give priority to `open` command.

Reply:
![image](https://github.com/reactos/reactos/assets/2107452/70d030fa-8019-4753-a72d-42004786d28f)